nearmint/Cargo.toml

```diff
@@ -4,12 +4,16 @@ version = "0.1.0"
 authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2018"
 
+[[bin]]
```

Comment:
This should not be an official entry point. If we want some entry point for testing then we can create `bin` folder under `testlib/src`. See for example `chain/devnet`.